_layouts/default.html

```diff
@@ -1,27 +1,19 @@
 <!DOCTYPE html>
-<html lang="{{ page.lang | default: site.lang | default: "en" }}">
+<html class="bg-dark" lang="{{ page.lang | default: site.lang | default: "en" }}">
 
   {%- include head.html -%}
 
-  <body>
+  <body class="d-flex flex-column vh-100">
 
     {%- include header.html -%}
 
-    <div class="py-5">
-      <div class="container page-content">
-          {{ content }}
-      </div>
-    </div>
+    <main role="main" class="flex-grow-1 bg-black text-light">
+      {{ content }}
+    </main>
 
     {%- include footer.html -%}
 
     <script async src="https://www.googletagmanager.com/gtag/js?id={{ site.google_analytics }}"></script>
-    <script>
```

Comment:
Why remove GA?

Reply:
Whoops, didn't mean to! Will add it back!